TRANSLATING.md

```diff
@@ -0,0 +1,225 @@
+
+Run Github Desktop, and do the following steps:
+
+* On the Welcome Screen, click on **Sign in to GitHub.com**
+* Sign in by following the instructions. These may vary depending on whether you have logged into Github via a web browser already. If necessary, click on **Authorize Desktop**.
+* Confirm access by entering your Github password.
+* for Mac, if the browser requests to open "Github Desktop.app", click **Allow**.
+* In Configure Git, enter your name and email address. These will be used to identify commits you make to Git. Click **Continue**.
+* Agree or decline to submit periodic usage stats, and click **Finish**.
+* Either:
+  - Select **Clone a Repository from the Internet** and enter `yourusername/jamulus`, or
+  - Select your own fork of Jamulus from the list (`yourusername/jamulus`) and click the **Clone** button.
+* Select the Local Path where the project should be stored, and click **Clone**.
+* It will display the page **Cloning jamulus** with progress indication.
+* On completion of cloning, it will ask "How are you planning to use this fork?". Answer "To contribute to the parent project".
+
+### Qt Linguist
+
+Qt Linguist is a part of the Qt development suite, and may either be installed via your Operating System's packaging manager,
+or by downloading from the [Qt Open Source download page](https://www.qt.io/download-open-source). In the latter case you
+will need to create an account on the Qt website.
+
+Under Linux
+* On RedHat or CentOS: `yum install qt5-linguist` or `dnf install qt5-linguist`
+* On Debian or Ubuntu: `apt install qttools5-dev-tools`
+
+Instructions for use are in the [Qt Linguist Manual](https://doc.qt.io/qt-5/qtlinguist-index.html)
+
+----
+## Doing a translation
+
+### Get the most up-to-date files and set up a new branch
+
+#### With git command line tools
+
+The first step is to get the local repo up to date with the upstream master:
+
+```
+cd projectdir/jamulus
+git fetch upstream
+```
+
+This fetches information about the current state of the upstream repo. It is now necessary to apply any upstream changes to the local repo.
+As the user will not be updating the `master` branch themselves, `git rebase` can be used to fast-forward to the current state:
+
+```
+git checkout master
+git rebase upstream/master
+git push
+```
+
+Finally, create a new branch for the changes that will be done (do not just do them on `master`):
+
+```
+git checkout -b v370-german
+```
+
+(The branch name is chosen by the user; the above name v370-german is an example for the German translation of V3.7.0)
+
+The branch will be used later as the source of a Pull Request.
+
+#### With Github Desktop
+
+Select the current repository as `jamulus`, and the current branch as `master`.
+
+Click on "Fetch origin" to get fully up to date with upstream.
+
+Drop down the "Current Branch" menu and click on the **New Branch** button. Give the branch a name such as `v370-german` and click **Create Branch**.
```

Comment:
I'm wondering if we should use a consistent/different branch name. The official tags are r3_7_0. So maybe we should aim to match that style? Seperator-less version numbers will start to look strange if we ever have a 3.7.10 or 3.10.7.

```suggestion
Drop down the "Current Branch" menu and click on the **New Branch** button. Give the branch a name such as `translate-r3_7_0-german` and click **Create Branch**.
```

Note: There are other places below which use the branch name, so if it is changed, it should be changed there, too.